packages/react-server-dom-relay/src/ReactFlightDOMRelayServerHostConfig.js

```diff
@@ -146,4 +147,8 @@ export function writeChunk(destination: Destination, chunk: Chunk): boolean {
 
 export function completeWriting(destination: Destination) {}
 
-export {close} from 'ReactFlightDOMRelayServerIntegration';
+export {close};
+
+export function closeWithError(destination: Destination, error: mixed): void {
```

Comment:
What are your plans for the `error` param passed to `closeWithError`?

Reply:
This is used by other host configs. Notably Node.js streams accepts an error passed when forcing a stream to close:

https://github.com/facebook/react/blob/master/packages/react-server/src/ReactServerStreamConfigNode.js#L77